frontend/src/components/misc/ConfigList.tsx

```diff
@@ -93,32 +94,89 @@ export function ConfigList({ configEntries, valueDisplay, renderTooltip }: { con
         columns.removeAll(x => x.dataIndex == 'type');
     }
 
-    return (
-        <KowlTable className={styles.configEntryTable}
+    const tableColumns: ColumnDef<ConfigEntry>[] = [
+        {
+            header: 'Configuration',
+            accessorKey: 'name',
+            cell: ({row: {original: record}}) => {
+                let name = (
+                    <Flex className={styles.nameText}>
+                        {record.name}
+                    </Flex>
+                );
+                if (renderTooltip) name = renderTooltip(record, name);
 
-            dataSource={configEntries}
-            columns={columns}
+                const sensitive = record.isSensitive && (
+                    <Tooltip label="Value has been redacted because it's sensitive" placement="top" hasArrow>
+                        <EyeInvisibleTwoTone twoToneColor={colors.brandOrange} />
+                    </Tooltip>
+                );
 
-            pagination={{
-                visible: false,
-                defaultPageSize: 10000,
-            }}
+                return (
+                    <div className={styles.name}>
+                        {name}
+                        <span className={styles.configFlags}>{sensitive}</span>
+                    </div>
+                );
+            }
+        },
+        {
+            header: 'Value',
+            accessorKey: 'value',
+            cell: ({row: {original: record}}) => <span className={styles.value}>{formatConfigValue(record.name, record.value, valueDisplay)}</span>
+        },
+        {
+            header: 'Type',
+            size: 120,
+            accessorKey: 'type',
+            cell: ({row: {original: {type}}}) => <span className={styles.type}>{type?.toLowerCase()}</span>
+        },
+        {
+            id: 'source',
+            header: () => (
+                <span className={styles.sourceHeader}>
+                Source
+                <Tooltip
+                    label={
+                        <>
+                            <p>Resources can be configured at different levels. Example: A topic config may be inherited from the static broker config.</p>
+                            <p>Valid sources are: Dynamic Topic, Dynamic Broker, Default Broker, Static Broker, Dynamic Broker Logger and Default config.</p>
+                        </>
+                    }
+                    placement="left"
+                    hasArrow
+                >
+                    <InfoCircleFilled style={{color: '#bbbbbb'}}/>
```

Comment:
Maybe just use `grey.500`

Reply:
Yes, should use the built in color "names"